Sources/TuistKit/ProjectEditor/ProjectEditorMapper.swift

```diff
@@ -237,10 +237,7 @@ final class ProjectEditorMapper: ProjectEditorMapping {
         )
         let scheme = Scheme(name: projectName, shared: true, buildAction: buildAction, runAction: runAction)
         let projectSettings = Settings(
-            base: [
-                "ONLY_ACTIVE_ARCH": "YES",
-                "EXCLUDED_ARCHS": .string(excludedArchs()),
-            ],
```

Comment:
@luispadron what was the problem with this? I think it's fine to only use the current arch for tuist edit

Reply:
Maybe but I was running into the same arch not defined issues because of this. In general since Tuist is universal it should be buildable with both archs, so Tuist edit shouldn't exclude imo.